test/e2e_node/mirror_pod_test.go

```diff
@@ -62,7 +62,7 @@ var _ = framework.KubeDescribe("MirrorPod", func() {
 			Testname: Mirror Pod, update
 			Description: Updating a static Pod MUST recreate an updated mirror Pod. Create a static pod, verify that a mirror pod is created. Update the static pod by changing the container image, the mirror pod MUST be re-created and updated with the new image.
 		*/
-		framework.ConformanceIt("should be updated when static pod updated [NodeConformance]", func() {
+		It("should be updated when static pod updated [NodeConformance]", func() {
```

Comment:
Mirror pod tests do not belong in conformance. Static pods require reconfiguring kubelets, which make it not an API feature.